build.sbt

```diff
@@ -37,6 +37,7 @@ def common: Seq[Setting[_]] = releaseSettings ++ bintraySettings ++ evictionSett
   licenses := Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0.html"))),
   homepage := Some(url("https://www.lagomframework.com/")),
   sonatypeProfileName := "com.lightbend",
+  Test / classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.ScalaLibrary,
```

Comment:
That might be a good addition to the build in `master`. 

I will forward port if it works well here.

Reply:
For context...

Since the upgrade to Scala 2.12.13, we are consistently getting this error in Travis:

```
[error] Caught java.lang.OutOfMemoryError: Metaspace
[error] To best utilize classloader caching and to prevent file handle leaks, we recommend 
running sbt without a MaxMetaspaceSize limit.  If this error occurred during a test or run 
evaluation, it can be caused by the choice of ClassLoaderLayeringStrategy. Of the available 
strategies, ClassLoaderLayeringStrategy.ScalaLibrary will typically use the least metaspace. 
```

Reply:
Seems that this is not enough or I didn't add it to enough places.

Anyway, I will continue as a gardening task on Monday.